Business_Coaching_Platform/Business_Coaching_Platform/settings.py

```diff
@@ -141,10 +152,3 @@
 LOGIN_URL = 'login'
 
 MEDIA_ROOT = os.path.join(BASE_DIR, 'static/images')
-
-REST_FRAMEWORK = {
```

Comment:
Why is this removed?

Reply:
Because it was already added in settings.py (line 109)